portal/templates/redesign/teach_new/materials_new.html

```diff
@@ -103,101 +105,112 @@ <h1 class="banner__text--primary">Teaching Resources</h1>
             <img title="KS1" alt="KS1 icon" src='{% static "portal/img_new/stage_icon_ks1.png" %}'>
             <h4>Children will learn the basics of coding by programming a delivery van to drive around simple routes. They will sequence instructions and use simple repetition
                 commands in a visual programming language.</h4>
-            <button class="button--regular button--primary--positive">Download all KS1 resources</button>
+            <a href="{{ 'KS1.zip'|cloud_storage }}" class="button--regular button--primary--positive">Download all KS1 resources</a>
         </section>
         <h3>Sessions</h3>
         <p class="body-text">Complete lesson plans for each session, including learning objectives, a list of useful resources, pupil activities, and guideance on share and
             review following the session.</p>
         <table class="table--resources">
             <tr>
                 <td><p class="body-text"><strong>Session 1</strong></p></td>
-                <td><p class="body-text"><a>Unplugged algorithms for moving along a route</a></p></td>
+                <td><p class="body-text">
+                    <a href="{% url 'materials_viewer_new' pdf_name='ks1_session_1' %}">Unplugged algorithms for moving along a route</a></p></td>
```

Comment:
Close your tags at the same level of indent that you open them - so here your `</p></td>` should be on a new line, aligned with the opening tags.

Just pretend I've repeated this comment everywhere that's happened...